src/pages/patientView/mutation/column/CohortColumnFormatter.tsx

```diff
@@ -5,35 +5,93 @@ import {IColumnFormatterData} from "../../../../shared/components/enhancedReactT
 import Tooltip from 'rc-tooltip';
 import 'rc-tooltip/assets/bootstrap_white.css';
 import {MutationTableRowData} from "../../../../shared/components/mutationTable/IMutationTableProps";
-
+import {MutSigData, VariantCountData} from "../../PatientViewPage";
+import {Mutation} from "../../../../shared/api/CBioPortalAPI";
+import {VariantCount} from "../../../../shared/api/CBioPortalAPIInternal";
 
 export default class CohortColumnFormatter {
 
     public static renderFunction(data:IColumnFormatterData<MutationTableRowData>, columnProps:any) {
-        const qValue:number = CohortColumnFormatter.getQValue(data, columnProps.data);
+        const mutSigQValue:number|null = CohortColumnFormatter.getMutSigQValue(data, columnProps.mutSigData);
+        const variantCountData = CohortColumnFormatter.getVariantCountData(data, (columnProps.variantCountData as VariantCountData));
+        const freqViz = CohortColumnFormatter.makeCohortFrequencyViz(data, (columnProps.variantCountData as VariantCountData))
         return (
-            <Td key={data.name} column={data.name}>
-                <If condition={qValue !== null}>
-                    {CohortColumnFormatter.makeMutSigIcon(qValue)};
-                </If>
+            <Td key={data.name} column={data.name} value={variantCountData && variantCountData.numberOfSamplesWithMutationInGene}>
+                <div>
+                    <If condition={freqViz !== null}>
+                        {freqViz}
+                    </If>
+                    <If condition={mutSigQValue !== null}>
+                        {CohortColumnFormatter.makeMutSigIcon(mutSigQValue as number)}
+                    </If>
+                </div>
             </Td>
         );
     };
 
-    private static getQValue(data:IColumnFormatterData<MutationTableRowData>, mutSigData:any) {
+    private static getVariantCountData(data:IColumnFormatterData<MutationTableRowData>, variantCountData:VariantCountData) {
+        if (!variantCountData || !variantCountData.geneData || !data.rowData || data.rowData.length === 0) {
+            return null;
+        }
+        const entrezGeneId = data.rowData[0].entrezGeneId;
+        const hugoGeneSymbol = data.rowData[0].gene.hugoGeneSymbol;
+        const keyword = data.rowData[0].keyword;
+
+        const numberOfSamples = variantCountData.numberOfSamples;
+        const geneData = variantCountData.geneData[entrezGeneId];
+
+        let numberOfSamplesWithMutationInGene, numberOfSamplesWithKeyword;
```

Comment:
With the current lint settings, multiple variable declarations in the same statement are not allowed. But, we can discuss this one too.